optimum/intel/__init__.py

```diff
@@ -191,6 +191,7 @@
             "OVModelOpenCLIPVisual",
             "OVModelOpenCLIPText",
             "OVModelOpenCLIPForZeroShotImageClassification",
+            "OVModelForZeroShotImageClassification",
```

Comment:
well, I can't find AutoModelOpenCLIPForZeroShotImageClassification. How it is loaded? I thought the concept of moving to OpenVINO backend is always replacing AutoModel to OvModel.

Reply:
AutoModel is transformers library integration way, this class for different library https://github.com/mlfoundations/open_clip it has different API. Models in such format are also available on huggingface https://huggingface.co/docs/hub/open_clip

I do not think that this PR is good place for such discussion and I do not think that removing open clip integration is good idea, not all models can be loaded by transformers as well that not all transformers supported clip models are compatible with open_clip